packages/runtime/container-runtime/src/opTelemetry.ts

```diff
@@ -49,7 +49,7 @@ export class OpTracker {
         });
 
         deltaManager.on("op", (message: ISequencedDocumentMessage) => {
-            this._nonSystemOpCount += isSystemMessage(message) ? 0 : 1;
+            this._nonSystemOpCount += !isClientMessage(message) ? 0 : 1;
```

Comment:
I think the default everywhere should be isSystemMessage -> !isRuntimeMessage.
If you look at current definitions of these functions in protocol, you will see that isSystemMessage !== !isSystemMessage